src/SignalR/common/Http.Connections/src/ConnectionOptionsSetup.cs

```diff
@@ -13,6 +13,7 @@ public class ConnectionOptionsSetup : IConfigureOptions<ConnectionOptions>
     /// <summary>
     /// Default timeout value for disconnecting idle connections.
     /// </summary>
+    /// <remarks>That's a known typo issue, while we think it's not worth worth making a breaking change here, see https://github.com/dotnet/aspnetcore/pull/30558#discussion_r585189841</remarks>
```

Comment:
Does this need to be a /// comment? IIRC, the XML docs are pulled through into learn.microsoft.com, and this doesn't seem like the sort of comment needed there, as it's more of a note for the code?

Reply:
At the start of the body seems fine.
Although @carlossanlop  is there an XML tag for "not to appear in docs, dev comment only"?

Reply:
@danmoseley 

- Remarks do not show up in the VS intellisense. They only show up in Microsoft Learn / MS Docs.
- If you truly want a dev comment, and don't even want it to show up in remarks in Microsoft Learn / Microsoft Docs, use double slash `//` instead of triple slash `///`.

Reply:
Ok. I've think it's clearest to just put a regular comment in the body here.

Reply:
> At the start of the body seems fine.

@danmoseley currently, the code comment is before the document comment, should I move it after the document comment and before the property declaration line?